lib/events/recorder.go

```diff
@@ -142,6 +142,10 @@ func (r *ForwardRecorder) GetAuditLog() IAuditLog {
 	return r.AuditLog
 }
 
+func (r *Forwarder) EmitAuditEvent(ctx context.Context, event AuditEvent) error {
+	return nil
+}
+
```

Comment:
Add comments describing why this method is non-functional.  Methods which expect never to be called should return `NotImplemented` errors.